app/views/liquids/_stackexchange.html.erb

```diff
@@ -18,11 +18,11 @@
         </div>
       </div>
       <a class="ltag__stackexchange--score-container" href="<%= post_url %>" target="_blank" rel="noopener noreferrer">
-        <img src="<%= ActionController::Base.helpers.asset_path("stackexchange-arrow-up.svg") %>" alt="">
+        <img src="<%= ActionController::Base.helpers.asset_path("stackexchange-arrow-up.svg") %>" alt="Up vote">
```

Comment:
```suggestion
        <img src="<%= ActionController::Base.helpers.asset_path("stackexchange-arrow-up.svg") %>" alt="">
```
I'm not sure we want to introduce an alt here or below, as it will change the link name to "Up vote <score> Down vote" 🤔